build_library/prod_image_util.sh

```diff
@@ -76,7 +77,7 @@ create_prod_image() {
 L+  /etc/ld.so.conf     -   -   -   -   ../usr/lib/ld.so.conf
 EOF
 
-  # Only try to disable rw on /usr if there is a /usr partition 
+  # Only try to disable rw on /usr if there is a /usr partition
```

Comment:
How about we add some 'read-only' flag to finish_image so there is no need to mount the ESP again. That would avoid all those changes to disk_util too.